dosfstools.yaml

```diff
@@ -0,0 +1,55 @@
+# Generated from https://git.alpinelinux.org/aports/plain/main/dosfstools/APKBUILD
+package:
+  name: dosfstools
+  version: "4.2"
+  epoch: 0
+  description: DOS filesystem utilities
+  copyright:
+    - license: GPL-3.0-or-later
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - libtool
+      - linux-headers
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 64926eebf90092dca21b14259a5301b7b98e7b1943e8a201c7d726084809b527
+      uri: https://github.com/dosfstools/dosfstools/releases/download/v${{package.version}}/dosfstools-${{package.version}}.tar.gz
```

Comment:
best practice is to use git-checkout, especially when upstream comes from github already. In which case autoreconf may be needed to build the package.

Reply:
@dustinkirkland